### ISSUE TITLE
Error when exporting to CASA gain table

### ISSUE THREAD

**Issue by SpheMakh** [ratt-ru/CubiCal#194]
```
- 18:46:36 - casa_db_adaptor    [io] [6.6/22.7 41.8/57.5 33.2Gb] Exporting to CASA gaintables
 - 18:46:36 - main               [io] [6.6/22.7 41.8/57.5 33.2Gb] I/O handler for load None save -1 failed with exception: RegularFileIO: error in open or create of file /home/makhathini/msdir/NGC4993_OFF_April18-D2-1GC-0.ms/G-field_0-ddid_None.parmdb.B.casa/table.f0i: No such file or directory
 - 18:46:36 - main               [io] [6.6/22.7 41.8/57.5 33.2Gb] Traceback (most recent call last):
  File "/cubical/cubical/workers.py", line 415, in _io_handler
    solver.gm_factory.close()
  File "/cubical/cubical/machines/abstract_machine.py", line 793, in close
    db.close()
  File "/cubical/cubical/database/casa_db_adaptor.py", line 454, in close
    self.__export()
  File "/cubical/cubical/database/casa_db_adaptor.py", line 439, in __export
    casa_caltable_factory.create_B_table(self, "G:gain")
  File "/cubical/cubical/database/casa_db_adaptor.py", line 231, in create_B_table
    viscal_label="B Jones" if diag else "D Jones")
  File "/cubical/cubical/database/casa_db_adaptor.py", line 52, in init_empty
    with tbl("%s::ANTENNA" % filename, ack=False, readonly=False) as t:
  File "/usr/local/lib/python2.7/dist-packages/casacore/tables/table.py", line 363, in __init__
    Table.__init__(self, tabname, lockopt, opt)
RuntimeError: RegularFileIO: error in open or create of file /home/makhathini/msdir/NGC4993_OFF_April18-D2-1GC-0.ms/G-field_0-ddid_None.parmdb.B.casa/table.f0i: No such file or directory
 - 18:46:36 - main               [0.4/0.6 3.3/3.4 33.2Gb] Exiting with exception: RuntimeError(RegularFileIO: error in open or create of file /home/makhathini/msdir/NGC4993_OFF_April18-D2-1GC-0.ms/G-field_0-ddid_None.parmdb.B.casa/table.f0i: No such file or directory)
 Traceback (most recent call last):
  File "/cubical/cubical/main.py", line 360, in main
    stats_dict = workers.run_process_loop(ms, tile_list, load_model, single_chunk, solver_type, solver_opts, debug_opts)
  File "/cubical/cubical/workers.py", line 205, in run_process_loop
    return _run_multi_process_loop(ms, load_model, solver_type, solver_opts, debug_opts)
  File "/cubical/cubical/workers.py", line 285, in _run_multi_process_loop
    ms.update_flag_counts(io_futures[-1].result()['flagcounts'])
  File "/usr/local/lib/python2.7/dist-packages/concurrent/futures/_base.py", line 455, in result
    return self.__get_result()
  File "/usr/local/lib/python2.7/dist-packages/concurrent/futures/_base.py", line 414, in __get_result
    raise exception_type, self._exception, self._traceback
RuntimeError: RegularFileIO: error in open or create of file /home/makhathini/msdir/NGC4993_OFF_April18-D2-1GC-0.ms/G-field_0-ddid_None.parmdb.B.casa/table.f0i: No such file or directory
```

**Comment by o-smirnov** [comment]
Fixed on maddermax branch. PR coming soon.

**Comment by o-smirnov** [comment]
Fixed by #196.
